tracer/samples/IISInDocker/Dockerfile

```diff
@@ -15,24 +15,16 @@ RUN msbuild /p:Configuration=Release
 
 ### Stage: runtime
 FROM mcr.microsoft.com/dotnet/framework/aspnet:4.8-windowsservercore-ltsc2019 AS runtime
-SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
-
 WORKDIR /inetpub/wwwroot
 COPY --from=build /app/src/. ./
-COPY startup.ps1 .
+COPY install-dd-trace-dotnet.ps1 startup.ps1 ./
+
+# Install the Datadog SDK for .NET using the msi installer
+RUN .\install-dd-trace-dotnet.ps1
+
+# Remove the default ENTRYPOINT "C:\ServiceMonitor.exe w3svc"
+ENTRYPOINT []
```

Comment:
Why remove the entrypoint and use cmd instead? 🤔 We would basically never want to change the cmd for this dockerfile otherwise it won't be configured correctly, so why make that easier to do accidentally?

Reply:
The `ENTRYPOINT` added by the base image is `C:\ServiceMonitor.exe w3svc`, but we need to call `startup.ps1` instead to add some last-minute env vars (which cannot be added earlier). That script then calls `C:\ServiceMonitor.exe w3svc` as the last step.

The previous version of this was also overriding the `ENTRYPOINT` for the same reason, but I think it makes more sense to call `startup.ps1` from `CMD` than from `ENTRYPOINT`, so I had to clear out the `ENTRYPOINT`.

Reply:
> I think it makes more sense to call `startup.ps1` from `CMD` than from `ENTRYPOINT`

The reasoning here is if you want to run a custom command, you override `CMD`. If you have an `ENTRYPOINT` defined, then what you pass to `CMD` just becomes _arguments_ to `ENTRYPOINT`, which would not work here.

I'll test this again to be sure, though.

Reply:
Although, now that I think about it, the base image was using `ENTRYPOINT`, so why mess with that? I'll change this, thanks.

Reply:
Done!

Reply:
This is so confusing: https://docs.docker.com/reference/dockerfile/#understand-how-cmd-and-entrypoint-interact